src/Constants.sol

```diff
@@ -0,0 +1,103 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.19;
+
+abstract contract Constants {
+  // Addresses
+  // Defender
+  address internal constant ETHEREUM_DEFENDER_ADDRESS = 0xA2A8BccD38138f1169ADdb0f3df9236a3CCCd753;
+  address internal constant OPTIMISM_DEFENDER_ADDRESS = 0xCeA11E14067697C085e1142afd2540b23f18304D;
+
+  // MessageExecutor
+  address internal constant ERC5164_EXECUTOR_OPTIMISM = 0x890a87E71E731342a6d10e7628bd1F0733ce3296;
+
+  // Multisigs
+  address internal constant EXECUTIVE_TEAM_OPTIMISM_ADDRESS =
+    0x8d352083F7094dc51Cd7dA8c5C0985AD6e149629;
+
+  // Chain IDs
+  uint256 internal constant ETHEREUM_CHAIN_ID = 1;
+  uint256 internal constant OPTIMISM_CHAIN_ID = 10;
+
+  // Deploy parameters
+  // Chainlink VRF
+  uint32 internal constant CHAINLINK_CALLBACK_GAS_LIMIT = 1_000_000;
+  uint16 internal constant CHAINLINK_REQUEST_CONFIRMATIONS = 3;
+
+  // Claimer
+  uint256 internal constant CLAIMER_MIN_FEE = 0.0001e18;
+  uint256 internal constant CLAIMER_MAX_FEE = 1000e18;
+
+  // Prize Pool
+  uint8 internal constant TIER_SHARES = 100;
+  uint8 internal constant RESERVE_SHARES = 100;
+  uint24 internal constant GRAND_PRIZE_PERIOD_DRAWS = 12;
+  uint32 internal constant DRAW_PERIOD_SECONDS = 4 hours;
+  uint256 internal constant MIN_TIME_AHEAD = DRAW_PERIOD_SECONDS;
+
+  // RngAuctions
+  // two auctions should end at the latest halfway through the draw period
+  uint64 internal constant AUCTION_DURATION = DRAW_PERIOD_SECONDS / 4;
+  uint64 internal constant AUCTION_TARGET_SALE_TIME = AUCTION_DURATION / 4;
+
+  // Twab
+  // nice round fraction of the draw period
+  uint32 internal constant TWAB_PERIOD_LENGTH = 1 hours;
```

Comment:
Looking at our activity: 

https://optimistic.etherscan.io/token/0x62BB4fc73094c83B5e952C2180B23fA7054954c4

I bet we could do it as infrequently as 4 hours?  Thoughts?

Reply:
We may experience a lot of activity at the beginning and also during promotional campaigns.
With 4 hours, we will be able to store more observations but do we really need it?